### DISCUSSION TITLE
How to convert waters data into the correct mzXML format for GNPS

### DISCUSSION THREAD

**Discussion by ttiantiann** [CCMS-UCSD/GNPS_Workflows#797] [Q&A]
Our lab use Waters UPLC/Xevo G2-XS QTOF to analyze the secondary metabolites of endophytic fungi, and wants to build an analysis network through these data to analyze the differences between metabolites.Instrument parameter settings are shown in Table 1. However, there are errors in converting to mzXML format. I have tried two methods: 
1. MS convert (Figure 1), and the conversion failed; 
2. GNPS_ Vendor_ Conversion (Figure 2). Although this method is successfully transformed, there are errors in building the molecular network (Figure 3).
Related tables and pictures are shown in the file.
Hope someone can help solve the problem. Look forward to your reply. Thank you so much!
[Summary of instrument parameters and errors.docx](https://github.com/CCMS-UCSD/GNPS_Workflows/files/7618095/Summary.of.instrument.parameters.and.errors.docx)

**Comment by mwang87** [comment]
Oh man waters data is a pain. @kyobinkang would be the best resource.

**Reply by ttiantiann** [reply]
Thanks for your help!

**Comment by KyobinKang** [comment]
There are a lot of possible reasons, so I cannot say anything without seeing your raw data. Key information is not included in the instrument parameter list; e.g. was it acquired in DDA mode or MSe mode?

For general guideline for Waters data conversion, see [this GNPS documentation page](https://ccms-ucsd.github.io/GNPSDocumentation/fileconversion_waters/)

**Reply by ttiantiann** [reply]
Thank you for your reply.It was required in MSe mode.I have tried FBMN to build a molecular network, but it seems that two or more data cannot compare their differences through FBMN.I want to send you two raw data and please help to see what the problem is ? Thanks a lot for your help!
PS：Can you send me your email address? Data files cannot be uploaded here.

**Reply by KyobinKang** [reply]
Quant-based comparison is one of the advantages of using FBMN; have you put the metadata into the FBMN analysis? 
+ I want to make it sure that you used MS-DIAL for the preprocessing of MSe data.

**Reply by ttiantiann** [reply]
Yes,I use MS-DIAL for the preprocessing of MSe data.I didn't upload the metadata into the FBMN.The following PDF is the steps for me to use MS-DIAL to do FBMN. I think this may better explain my problem.Please point out my mistakes when you are free.🙏
[Use MS-DIAL to export result for FBMN.pdf](https://github.com/CCMS-UCSD/GNPS_Workflows/files/7623441/Use.MS-DIAL.to.export.result.for.FBMN.pdf)

**Reply by KyobinKang** [reply]
You need to submit one MGF file with one feature table. For comparison between multiple samples, you need to process all the data together and put them together via alignment step in MS-DIAL. Then, your MGF and feature table will contain MS features from every files. Group the files by submitting the metadata. It's done.

**Reply by ttiantiann** [reply]
Get it,thanks for your answer!
